app/controllers/likes_controller.rb

```diff
@@ -4,7 +4,15 @@ def create
     @post = @user.posts.find(params[:post_id])
     @like = @user.likes.new(post_id: @post.id, author_id: @user.id)
 
-    redirect_to user_post_path(@user, @post) if @like.save
+    # @post.save
+    # redirect_to user_posts_path(@user)
```

Comment:
### Optional
- [ ] Kindly delete all commented pieces of code, it makes our codebase messy and difficult to understand